src/assets/mixins.styl

```diff
@@ -1,16 +1,29 @@
 //fonts mixins
-play($weight)
-  font-family 'Playfair Display', serif
-  font-weight $weight
-
-
-source($weight)
-  font-family 'Source Sans Pro', sans-serif
+setFont($family, $weight)
+  if $family == 'playfair'
+    font-family 'Playfair Display', serif
+  if $family == 'source'
+    font-family 'Source Sans Pro', sans-serif
   font-weight $weight
 
 //Grid mixin
 makeGrid($cols, x=25px, y=25px)
   grid-template-columns repeat($cols, 1fr)
   grid-template-rows auto
   grid-gap x y
-  
\ No newline at end of file
+  
+//media quieries mixin  
+mediaQ($size)
```

Comment:
leave it in "media-query" "respond-to" or something more with dash instead of camelcase

Reply:
and use it, you created the mixin but you are not using in all across the files

Reply:
I'm naming it with camelcase because is being used in the name of the other mixins "setFont" and "makeGrid", should I leave it "mediaQuery" instead?